src/frontend/src/styles.css

```diff
@@ -43,30 +43,10 @@ table.scroll tbody {
   padding: 0;
 }
 
-@media screen and (max-width: 1023px) {
-  .navbar-menu.is-active {
-    display: flex !important;
-    padding: 0;
-  }
-}
-
 .app-navbar {
   box-shadow: 0 2px 3px hsla(0,0%,7%,.1), 0 0 0 1px hsla(0,0%,7%,.1);
 }
 
-.navbar-menu{
-  flex-grow: 1;
-  flex-shrink: 0;
-}
-.navbar-start{
-  justify-content: flex-start;
-  margin-right: auto;
-}
-.navbar-end{
-  justify-content: flex-end;
-  margin-left: auto;
-}
```

Comment:
I don't think we should remove these; and also should these move to navbar component css? 

api status text must be just before the burge menu